libs/estdlib/src/erlang.erl

```diff
@@ -294,6 +294,11 @@ process_info(_Pid, _Key) ->
 %%      <li><b>esp32_minimum_free_size</b> the smallest number of free bytes in the ESP32 heap since boot (integer)</li>
 %%      <li><b>esp32_chip_info</b> Details about the model and capabilities of the ESP32 device (map)</li>
 %% </ul>
+%% The following keys are supported on the STM32 platform:
+%% <ul>
+%%      <li><b>avm_free_heap_size</b> the number of (noncontiguous) free bytes in the STM32 heap (integer)</li>
+%%      <li><b>avm_minimum_free_size</b> the smallest number of free bytes in the STM32 heap since boot (integer)</li>
```

Comment:
Are we ok with avm prefix ? (instead of `atomvm_` used in `spawn_opt` ?)

Don't we want to use it for esp32 as well ?

Reply:
I think `atomvm_` is better. I was not sure if I should rename the esp32 keys now in this PR, or leave that for a separate PR after support for them has been added to RP2 platform. I guess it does make the most sense to rename the esp32 keys now, and add RP2 once an implementation for them is worked out.